packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -73,21 +115,24 @@ const ColumnHeaderCell = (props) => {
     <th
       key={column.id}
       className={cx('column-header', theme.className, { selectable: !(column.isSelectable === false) })}
-      tabIndex="-1"
+      tabIndex={isTabStop ? 0 : -1}
       role="columnheader"
       scope="col"
       aria-sort={column.sortIndicator}
-      onClick={(!(column.isSelectable === false) && onColumnSelect) ? onHeaderSelect : undefined}
-      style={{ width, height: headerHeight }}
+      onMouseDown={(!(column.isSelectable === false) && onColumnSelect) ? handleMouseDown : undefined}
```

Comment:
Why is the purpose of? It feels like a workaround.
```js
!(column.isSelectable === false)
```

Reply:
We want the default behavior to be that cells are selectable.  This code will allow a selectable state unless consumers pass false and maintains passivity with existing API.  The only other option would be to break passivity and create a prop like disableSelection.  However, this code was introduced as part of the introduction of the hover functionality.

Reply:
As discussed, we flattened the props and gave a default to the isSelectable prop. Therefore, we no longer have to compensate for undefined and can expect the value to be either true or false.